locale/en_US/manager.xml

```diff
@@ -429,6 +429,7 @@
 	<message key="settings.roles.roleOptions">Role Options</message>
 	<message key="settings.roles.showTitles">Show role title in contributor list</message>
 	<message key="settings.roles.permitSelfRegistration">Allow user self-registration</message>
+	<message key="settings.roles.recommendOnly">This role can not make a review decission, only a recommendation.</message>
```

Comment:
This role is only allowed to recommend a review decision and will require an authorised editor to record a final decision.